back/db/structure.sql

```diff
@@ -1627,9 +1627,10 @@ CREATE TABLE public.phases (
     manual_voters_amount integer,
     manual_voters_last_updated_by_id uuid,
     manual_voters_last_updated_at timestamp(6) without time zone,
-    user_fields_in_form boolean DEFAULT false NOT NULL,
+    survey_popup_frequency integer,
     similarity_threshold_title double precision DEFAULT 0.3,
-    similarity_threshold_body double precision DEFAULT 0.4
+    similarity_threshold_body double precision DEFAULT 0.4,
+    user_fields_in_form boolean DEFAULT false NOT NULL
 );
```

Comment:
Just want to double-check these changes are ok to merge into master? The survey_popup_frequency is from my FE feature branch I believe?

Reply:
This is the only thing that looks off to me, too.